db/migrate/20230906112602_create_user_languages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserLanguages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user_languages do |t|
+      t.references :user, foreign_key: true, null: false
+      t.string :language
```

Comment:
I suspect this is auto-generated by ActiveRecord this way, but to my eye this seems like a field that should be `null: false` and enforced by Postgres?

Reply:
I had a feeling that I forgot something.